app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3:
+        raise ValueError("Invalid command format")
+
+    cp, source_file, destination_file = parts
```

Comment:
source_file, destination_file is not true naming
it is path. Use source_file_path or source_path
The same with destination

Reply:
I see
will upload changes now